docs/release-notes/0.10.1.md

```diff
@@ -0,0 +1,4 @@
+### 0.10.1 {small}`the future`
+
+```{rubric} Bugfix
+```
```

Comment:
Can we hold off on putting 0.10.1 file before we do a 0.10 release? I don't see any value in the 0.10.0 docs have and empty 0.10.1 section.

Reply:
sure!